src/app/page.tsx

```diff
@@ -1,9 +1,11 @@
 import Header from "@/components/home/header";
+import SeeMoreButton from "@/components/home/seemore";
 
 const Home = () => {
   return (
-    <div>
+    <div className="flex flex-col items-center">
```

Comment:
```suggestion
    <div>
```
we will not use styling on this page, they should be in your component